### DISCUSSION TITLE
When to use Issues vs Discussions

### DISCUSSION THREAD

**Discussion by oligocene** [glzr-io/glazewm#765] [General]
I am actually a little confused.
When should I use the one vs the other?

V.3.3.0 is a buggy mess on my laptop, and on a couple of my VMs.
When switching to ws's with VSCode the screen goes black, same on ws's with Directory Opus and couple of ither apps.
Is this an Issues-case or Discussions?

I've rolled back everything to 3.2 excpet from one VM.
The new "remove title bar" seems to nuke most of my setups.

**Comment by lars-berger** [comment]
Is the black screen bug only happening with the `hide_title_bar` option enabled? Also noticing that some apps really don't play nicely with it enabled

Rule of thumb is gh issues to report bugs or suggest specific features - everything else on gh discussions or discord

**Comment by oligocene** [comment]
Thanks for the clarification.

The black screen bug happended with the option enabled. I then exited GlazeWM, switched the option off and started GlazeWM.
But, the problem was still there. After stopping and starting apps, switching between workspaces (just trying different things) it seems like my graphics driver crashed (Lenovo laptop w/Radeon 600M). That was it, reboot and roll back to 3.2.

Yeah, I've been playing around with removing the title bar in the past. It always brought a certain level of "mess", some apps do not like this at all.

**Reply by lars-berger** [reply]
> Thanks for the clarification.
> 
> The black screen bug happended with the option enabled. I then exited GlazeWM, switched the option off and started GlazeWM. But, the problem was still there. After stopping and starting apps, switching between workspaces (just trying different things) it seems like my graphics driver crashed (Lenovo laptop w/Radeon 600M). That was it, reboot and roll back to 3.2.
> 
> Yeah, I've been playing around with removing the title bar in the past. It always brought a certain level of "mess", some apps do not like this at all.

We haven't added proper cleanup for the 2 new window effects (title bar + corner style), so some windows will be left modified after exit. Leaving the setting disabled should be the same behavior as before
